.env.devnet

```diff
@@ -0,0 +1,8 @@
+REACT_APP_RPC_URL_SHARD0=https://api.s0.ps.hmny.io/
+REACT_APP_RPC_URL_SHARD1=https://api.s1.ps.hmny.io/
+REACT_APP_RPC_URL_SHARD2=https://api.s2.t.hmny.io/
```

Comment:
can we leave those shard2/shard3 URL empty ? will it cause any issue to the UI If they are ? I am thinking if a wallet exist on both mainnet and devnet, the devnet explorer will then show the mainnet balance of that account in s2 and s3

Reply:
we can leave them empty and not use them (there is an argument telling the explorer which shards to use and its configured to 0, 1 for now)

I'll push a change up there so devnet files will have those 2 empty